etcdserver/server.go

```diff
@@ -256,7 +258,7 @@ func NewServer(cfg *ServerConfig) (*EtcdServer, error) {
 		}
 
 		if cfg.ShouldDiscover() {
-			log.Printf("etcdserver: discovery token ignored since a cluster has already been initialized. Valid log found at %q", cfg.WALDir())
+			plog.Warningf("discovery token ignored since a cluster has already been initialized. Valid log found at %q", cfg.WALDir())
```

Comment:
If we encourage people to remove the bootstrap flags when restarting etcd, we should warn them here. If people are allowed to use the same command line, it is just a info, and people don't need to change the command line. Does this make sense?